src/material.cpp

```diff
@@ -60,8 +60,8 @@ namespace {
 
   // QueenMinorsImbalance[opp_minor_count] is applied when only one side has a queen.
   // It contains a bonus/malus for the side with the queen.
-  const int QueenMinorsImbalance[16] = { 
-    31, -8, -15, -25, -5, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0
+  const int QueenMinorsImbalance[PIECE_NB] = {
```

Comment:
I used PIECE_NB because it was 16 in the original code. We could nitpick and write [12], no strong opinon on this, please tell me if  you want me to define QueenMinorsImbalance[12]